plaso/preprocessors/macos.py

```diff
@@ -244,7 +242,7 @@ def _GetPlistRootKey(self, file_entry):
       file_entry (dfvfs.FileEntry): file entry of the plist.
 
     Returns:
-      plistlib._InternalDict: plist root key.
+      dict[str, object]: plist root key.
```

Comment:
seeing this dict represents a plist key what does it contain?

Reply:
A mapping of string keys to plist values. See https://docs.python.org/3/library/plistlib.html. This is common for all plist handling code in Plaso.

Reply:
I don't know what you want here. This is a dictionary mapping string keys to plist values. See https://docs.python.org/3/library/plistlib.html

Reply:
ack I thought there might be specific convention but does not appears to be so